### DISCUSSION TITLE
Have to restart Docker container when modifying public folder volume

### DISCUSSION THREAD

**Discussion by shotor** [vercel/next.js#42075] [Help]
I have this issue and haven't been able to find any one else experiencing this.

I'm mapping a few directories as docker volumes into my nextjs public folder:

```docker
- ~/myimages:/app/public/images
- ~/myaudio:/app/public/audio
```

It works fine, but when I add a file on my machine to `~/myimages` NextJS is unable to find it untill I restart the Docker container. I don't want to restart the Docker container, because I'm using SSG and it takes me back to the state when I built the Docker image.

Is there some configuration I'm missing here?

**Comment by chin8628** [comment]
If anyone faces the same problem, this stackoverflow thread explained it well.
https://stackoverflow.com/a/60977837/4122078
